src/internal.c

```diff
@@ -28070,7 +28069,20 @@ static int DoSessionTicket(WOLFSSL* ssl, const byte* input, word32* inOutIdx,
         if (!session) {
             WOLFSSL_MSG("Session lookup for resume failed");
             ssl->options.resuming = 0;
+            return ret;
+        }
+        #if defined(HAVE_SESSION_TICKET) || !defined(NO_PSK)
+        #ifdef WOLFSSL_TLS13
```

Comment:
It is for TLS1.2 and below. Need WOLFSSL_TLS13 definition since session->ticketSeen is defined under the macro. Wondering session->bornOn to be used instead.

Reply:
We can't have this only work when TLS 1.3 is compiled in.
There is inside and outside data. Use ssl->session.timeout for the lifetime hint.

Reply:
Agree. Rewrote them with session.bornOn and session.timeout.